site/src/pages/components/radio.research.mdx

```diff
@@ -10,26 +11,26 @@ import Anatomy from '../../components/anatomy'
 import Concepts from '../../components/concepts'
 import ComponentCoverage from '../../components/component-coverage'
 
-<ComponentCoverage component="Radio Button" />
+<ComponentCoverage component="Radio" />
 
 ## Introduction
 
-_(For a lengthier discussion, on which this page was based, please refer to the [original radio button issue](https://github.com/openui/open-ui/issues/251))_
+_(For a lengthier discussion, on which this page was based, please refer to the [original radio issue](https://github.com/WICG/open-ui/issues/251))_
 
-The research showed that different design systems treat radio buttons in one of two ways, i.e.:
+The research showed that different design systems treat radios in one of two ways, i.e.:
 
-- as a standalone component, that could be hosted on a “**radio button group**” component;
-- as an invariably non-independent part of a “**radio button group**” component (just like `<option>` tags are dependent on `<select>` tags).
+- as a standalone component, that could be hosted on a “**radio group**” component;
+- as an invariably non-independent part of a “**radio group**” component (just like `<option>` tags are dependent on `<select>` tags).
 
-The two approaches seem very similar at first, but the former seems to indicate that single “radio buttons” could exist, to probably indicate a binary state (on or off). The latter, however, shifts the scope up to the “radio button group”, which, although nonexistent as an HTML element, is an important concept for the W3 specification:
+The two approaches seem very similar at first, but the former seems to indicate that single “radios” could exist, to probably indicate a binary state (on or off). The latter, however, shifts the scope up to the “radio group”, which, although nonexistent as an HTML element, is an important concept for the W3 specification:
```

Comment:
let's remove _probably_ and be a bit more emphatic on your takeaways from the research. If you don't have a strong position then make it an open question.